scripts/testing/demonstrate_feature.sh

```
@@ -0,0 +1,73 @@
+
+echo "✅ File created: bad_demo.py"
+echo ""
+
+echo "2️⃣ Running quality analysis..."
+echo "----------------------------------------"
+./venv/bin/python scripts/ast_grep_final_analyzer.py bad_demo.py | grep -E "(Quality Score|Issues|print-call|nested-if|nested-loops)"
+echo ""
+
+echo "3️⃣ Simulating PostToolUse hook trigger..."
+echo "----------------------------------------"
+echo '{"tool_name": "Edit", "tool_input": {"file_path": "'$(pwd)'/bad_demo.py"}}' | ./venv/bin/python .claude/hooks/quality-check.py 2>&1
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)